src/main/java/org/spongepowered/api/command/ImmutableCommandMapping.java

```diff
@@ -36,7 +36,7 @@
  * An immutable command mapping instance that returns the same objects that
  * this instance is constructed with.
  */
-public final class ImmutableCommandMapping implements CommandMapping {
+public class ImmutableCommandMapping implements CommandMapping {
```

Comment:
`ImmutableModuleCommandMapping` is an `ImmutableCommandMapping` which has an associated `AbstractCommandModule`. While it is possible to implement `CommandMapping`, it would just result in a lot of duplicated code.